kubernetes-manifests/userservice.yaml

```diff
@@ -48,6 +48,12 @@ spec:
         # Valid levels are debug, info, warning, error, critical. If no valid level is set, gunicorn will default to info.
         - name: LOG_LEVEL
           value: "info"
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONTAINER_NAME
+          value: "userservice"
```

Comment:
As explained in this issue https://github.com/census-instrumentation/opencensus-python/issues/796#issuecomment-539109321
the stackdriver exporter needs these env vars explicitly declared